src/components/fin/metricSlider/waterTotalChart/WaveContent.tsx

```diff
@@ -0,0 +1,47 @@
+import Trans from '@/components/translation/Trans'
+import Wave from 'react-wavify'
+import { twMerge } from 'tailwind-merge'
+import Octopus from './waveContent/Octopus'
+
+type Props = {
+  isStatic?: boolean
+}
+export default function WaveContent({ isStatic }: Props) {
+  return (
+    <div
+      className={twMerge(
+        'relative overflow-hidden rounded-b-xl px-4 pb-6 pt-12 transition-opacity lg:px-36 lg:pb-8 lg:pt-14'
```

Comment:
Only one parameter here (maybe in other places I did not pay attention before :see_no_evil: )